src/main/kotlin/org/jetbrains/research/testspark/tools/kex/generation/KexProcessManager.kt

```diff
@@ -0,0 +1,350 @@
+                // remove imports from helper classes
+                imports.addAll(
+                    getImportsCodeFromTestSuiteCode(testCode, projectContext.classFQN!!)
+                        .filterNot {
+                            it.contains("import static org.example.EqualityUtils.*;") ||
+                                it.contains("import static org.example.ReflectionUtils.*;")
+                        },
+                )
+            }
+        } else {
+            kexErrorManager.errorProcess(
+                KexMessagesBundle.get("TestsDontExist"),
+                project,
+                errorMonitor,
+            )
+        }
+
+        ToolUtils.transferToIJTestCases(report)
+        ToolUtils.saveData(
+            project,
+            report,
+            packageStr,
+            imports,
+            projectContext.fileUrlAsString!!,
+            generatedTestsData,
+        )
+    }
+
+    private fun runKex(
+        cmd: MutableList<String>,
+        errorMonitor: ErrorMonitor,
+        indicator: CustomProgressIndicator,
+    ) {
+        try {
+            val kexProcess = GeneralCommandLine(cmd)
+            kexProcess.charset = Charset.forName("UTF-8")
+            kexProcess.workDirectory = File(kexHome)
+            kexProcess.environment["KEX_HOME"] = kexHome
+
+            val handler = OSProcessHandler(kexProcess)
+            val output = ProcessOutput()
+            ProcessTerminatedListener.attach(handler)
+
+            // rerouting stdout and stderr
+            handler.addProcessListener(object : ProcessAdapter() {
+                override fun onTextAvailable(event: ProcessEvent, outputType: Key<*>) {
+                    if (ToolUtils.isProcessStopped(errorMonitor, indicator)) {
+                        handler.destroyProcess()
+                        return
+                    }
+                    if (outputType.toString() == "stdout") {
+                        output.appendStdout(event.text)
+                    } else if (outputType.toString() == "stderr") {
+                        output.appendStderr(event.text)
+                    }
+                }
+
+                override fun processTerminated(event: ProcessEvent) {
+                    log.info("Process terminated with exit code: ${event.exitCode}")
+                    log.info("Output from Kex stdout:\n ${output.stdout}")
+                    log.info("Output from Kex stderr:\n ${output.stderr}")
+                }
+            })
+
+            handler.startNotify()
+
+            // Wait for the process to complete with a timeout
+            if (!handler.waitFor(kexProcessTimeout)) {
+                handler.destroyProcess()
+                throw IOException("Process timed out and was terminated")
+            }
+        } catch (e: IOException) {
+            e.printStackTrace()
+            log.error("Error running KEX process", e)
+        }
+    }
+
+    private fun getHelperClassBody(testCode: String) =
```

Comment:
We should make this code less reliant on the current structure of kex's tests:
1. We should ensure that we select correct class in `getHeleprClassBody` (e.g. by name)
2. We should ensure that `setup` and `test` methods are correctly identified in `extractTestMethod`, e.g. using annotations
3. In `extractTestMethod` we can also use annotations to correctly identify fields